twisted/conch/ssh/keys.py

```diff
@@ -1121,6 +1191,11 @@ def _toString_AGENTV3(self):
             elif self.type() == 'DSA':
                 values = (data['p'], data['q'], data['g'], data['y'],
                           data['x'])
+            # I assume this is what's needed
+            # But commented out because it hasn't been tested yet.
+            elif self.type() == 'EC':
+                raise UnsupportedAlgorithm("toString() does not support Elliptic Curves yet.")
```

Comment:
Whenever you have an unsupported thing like this, you should have a comment that links to a ticket on twistedmatrix.com, so that when people comes along they know what to do about it :)